### PR TITLE
Add prettier

### PR DESCRIPTION
**Pull request by marcusgchan**
## Description

- Added prettier
- Ran prettier
- Added prettier check for pipeline

## How to run prettier

- ```npm i``` to install prettier
- ```npm run format:check``` to check formatting
- ```npm run format:write``` to format
- Or setup prettier with you code editor
- If you're using vscode you can download the prettier extension and set it to format on save if you like

### CODE REVIEW

File: .prettierrc.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (KiaanCastillo) [inline .prettierrc.json:1]: This is great! Thanks Marcus. I don't see any rules in the `.prettierrc.json` file. Is it stored somewhere else?

Comment (KiaanCastillo) [reply .prettierrc.json:1]: Also, what do you think of these rules?
```
{
  "semi": false,
  "tabWidth": 2,
  "singleQuote": false,
  "jsxSingleQuote": false,
  "trailingComma": "es5",
  "bracketSpacing": true
}
```

Comment (marcusgchan) [reply .prettierrc.json:1]: I don't really have an opinion with the formatting rules, so i just left it blank for the defaults. Personally, I prefer having semi colons, but I'm cool either way.

Comment (marcusgchan) [reply .prettierrc.json]: Ok I added those rules and ran prettier to format the files

Comment (KiaanCastillo) [reply .prettierrc.json]: Okay nice, I am indifferent to the semicolons as well. I'm looking up some best practices and it looks like keeping semi is best

Comment (marcusgchan) [reply .prettierrc.json]: Ok I'll change semicolons to true and merge it

**Pull request merged**